### ISSUE TITLE
Activate HSV via MQTT

### ISSUE THREAD

**Issue by alexandreghez** [seydx/homebridge-camera-ui#479]
Hello,

It seems that HSV recording is triggered when videoAnalysis is set to true AND a motion is detected.

My camera have a MQTT feature thar triggers when motion is detected and i would like to know if the HSV recording could be trigerred with this MQTT message instead of the video analysis done by cameraUI.

With this solution, the VideoAnalysis could be set to false and avoid CPU / memory ?

**Comment by teymur1988** [comment]
Hi @alexandreghez,

You don't need videoanalysis enabled in order to record HSV. All is needed is to trigger the camera motion sensor and that would trigger the HSV recording. Doesn't matter how you trigger it, you can use http, mqtt, ftp, smtp - depending on what your camera can do. Videoanalysis is needed when your camera isn't capable of the protocols above then plugin will use low resolution stream (if one is available and configured) in order to process the frames and trigger motion if it thinks it's detected.

**Comment by alexandreghez** [comment]
You're right. I have an error because i set the same motion top for my 2 cameras...
